ncclient/devices/junos.py

```diff
@@ -110,3 +114,15 @@ def transform_reply(self):
             return reply
         else:
             return reply.encode('UTF-8')
+
+    def get_xml_parser(self, session):
+        # use_filter in device_params can be used to enabled using SAX parsing
+        if self.device_params.get('use_filter', False):
+            l = session.get_listener_instance(SAXParserHandler)
```

Comment:
You've made using the new SAX parser something that only Junos gets without writing another device handler. Any particular reason you didn't integrate into the default device handler? After all, you've refactored the code for all :)